priority_variables_transform/ARIC/hist_my_inf.yaml

```diff
@@ -733,6 +733,8 @@ class_derivations:
     slot_derivations:
       associated_participant:
         populated_from: phv00203858
+      age_at_condition_start:
+        expr: case(phv00295237 == 2, {phv00203852} * 365
```

Comment:
Both of these new expressions in this file are missing end parens ) and should be:
```
case(phv00295237 == 2, {phv00203852} * 365)
```
With the correct phv#'s of course.